src/handler/state.jl

```diff
@@ -1,8 +1,22 @@
+struct ChangedAsset
+    url ::String
+    modified ::Int
```

Comment:
@waralex Is `modified` a timestamp of some number of seconds since an earlier point in time?

Reply:
@rpkyle Just like in Python, this is the modify time of the file. I.e. this is the UNIX timestamp of the file change time